--- a/drw.c	2022-04-07 09:20:43.990154697 +0700
+++ b/drw.c	2022-04-07 09:20:28.000153598 +0700
@@ -203,6 +203,8 @@ drw_clr_create(Drw *drw, Clr *dest, cons
 	                       DefaultColormap(drw->dpy, drw->screen),
 	                       clrname, dest))
 		die("error, cannot allocate color '%s'", clrname);
+
+    dest->pixel |= 0xff << 24;
 }
 
 /* Wrapper to create color schemes. The caller has to call free(3) on the
